backend/src/v1/routes/report-routes.ts

```diff
@@ -0,0 +1,44 @@
+import express, { Request, Response } from 'express';
+import passport from 'passport';
+import { auth } from '../services/auth-service';
+import { reportService, enumReportStatus } from '../services/report-service';
+import HttpStatus from 'http-status-codes';
+import { utils } from '../services/utils-service';
+
+const reportRouter = express.Router();
+
+/**
+ * /api/v1/report/?status=<string>
+ *     Get all published reports for the company associated with the logged in user.
+ */
+reportRouter.get(
+  '/',
+  passport.authenticate('jwt', { session: false }, undefined),
+  auth.isValidBackendToken(),
```

Comment:
Sonar identified a possible bug here.  auth.isValidBackendToken() needs to be wrapped in a call to utils.asyncHandler(...).  See example in code-routes.ts.